omero/developers/Python.rst

```diff
@@ -222,6 +222,37 @@ Read data
             for image in dataset.listChildren():
                 print_obj(image, 4)
 
+-  **Get Objects by their ID or attributes**
+
+   The first argument for ``conn.getObjects()`` or ``conn.getObject()`` is the object type.
+   This is not case sensitive. Supported types are
+   ``project``, ``dataset``, ``image``, ``screen``, ``plate``, ``plateacquisition``, ``acquisition``, ``well``,
+   ``roi``, ``shape``, ``experimenter``, ``experimentergroup``, ``originalfile``, ``fileset``, ``annotation``
+
+::
+
+    # Find objects by ID
+    projects = conn.getObjects("Project", [1, 2, 3])
+
+    # Single object. IDs are unique across all types of annotations
```

Comment:
``IDs are unique across all types of annotations`` -> not sure what IDs are unique - is this a requirement for the lines I am supposed to write, or a given which comes from OMERO structuring of annotations ?